database/postgres/postgres.go

```diff
@@ -266,13 +277,23 @@ func runesLastIndex(input []rune, target rune) int {
 	return -1
 }
 
+func (p *Postgres) quoteIdentifier(name string) string {
+	if p.config.MigrationsTableHasSchema {
+		firstDotPosition := strings.Index(name, ".")
+		if firstDotPosition != -1 {
```

Comment:
Should we error if a `.` is not found?

Reply:
> Should we error if a . is not found?

@dhui  Why? I suggest to use default `schema` if `.` not found in table name.

Do you think this is a bad idea? Do you prefer that schema name must be mandatory?

Reply:
Falling back to `search_path` makes sense. The behavior isn't obvious based on the config name, so could you update the docs to mention that `.` is not required?